packages/react-components/react-popover/src/components/Popover/Popover.cy.tsx

```diff
@@ -541,4 +541,23 @@ describe('Popover', () => {
         .should('have.length', 2);
     });
   });
+
+  describe('popover not use transform', () => {
```

Comment:
I don't think this test is necessary in terms of positioning, it's really the visual result that matters at the end of the day, which is what the vr tests are for

Reply:
Removed the unnecessary test. thanks!